src/app-layout/visual-refresh-toolbar/index.tsx

```diff
@@ -202,10 +202,16 @@ const AppLayoutVisualRefreshToolbar = React.forwardRef<AppLayoutProps.Ref, AppLa
       fireNonCancelableEvent(onSplitPanelPreferencesChange, detail);
     };
 
+    const splitPanelDefaultSize = useMemo(
+      () => getSplitPanelDefaultSize(splitPanelPreferences?.position ?? 'bottom'),
```

Comment:
Should the same be done for classic and vr variants? Also, the getSplitPanelDefaultSize can be reused in all variants I think

Reply:
A fun fact is while answering your question I understood the real root cause of this problem: this "shaking" is caused by split panel animation for width. This bug reproduces only in refresh-toolbar theme, because refresh and classic don't have such an animation. 

This problem is solved by simply applying `overflow: hidden` for the split panel parent element.